core/templates/pages/exploration-editor-page/editor-tab/test-interaction-panel/test-interaction-panel.component.spec.ts

```diff
@@ -36,9 +38,18 @@ describe('Test Interaction Panel directive', function() {
 
   beforeEach(angular.mock.module('oppia'));
 
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule]
+    });
+  });
+
   beforeEach(angular.mock.module('oppia', function($provide) {
     $provide.value(
       'StateEditorRefreshService', TestBed.get(StateEditorRefreshService));
+    $provide.value(
```

Comment:
Any reason we can't use importAllAngularServices here?

Reply:
Maybe we can use it here. But this file had this structure so, I followed this. Also, this will be refactored anyways when migrated to angular.

Reply:
OK, sounds good.